packages/api/src/command/settings/updateSettings.ts

```diff
@@ -31,12 +31,12 @@ export const updateSettings = async ({
 };
 
 export type UpdateWebhookStatusCommand = {
-  id: string;
+  cxId: string;
   webhookEnabled: boolean;
   webhookStatusDetail?: string;
 };
 export const updateWebhookStatus = async ({
-  id,
+  cxId,
```

Comment:
As I was reviewing the WH code to remove the notification I remembered how it was confusing for someone's onboarding (I think yours!) WRT settings having an `id` column but that actually meaning the cx's ID. So I wend on and updated the parameters on the functions to better represent what that field really means.